openedx/features/ucsd_features/utils.py

```diff
@@ -0,0 +1,67 @@
+import json
+import logging
+from django.conf import settings
+from smtplib import SMTPException
+from django.core.mail import send_mail
+from django.template.loader import get_template
+from django.core.mail import EmailMultiAlternatives
+
+log = logging.getLogger(__name__)
+TEMPLATE_PATH = '{key}_email.html'
+
+
+def send_notification(key, data, subject, from_email, dest_emails):
+    """
+        Send an email.
+
+        params:
+            key - Email template will be selected on the basis of key
+            data - Dict containing context/data for the template
+            subject - Email subject
+            from_email - Email address to send email
+            dest_emails - List of destination emails
+        return:
+            boolean variable indicating email response.
+    """
+    try:
+        content = json.dumps(data)
+        email_template_path = TEMPLATE_PATH.format(key=key)
+        html_content = get_template(email_template_path).render(data)
+        msg = EmailMultiAlternatives(subject, content, from_email, dest_emails)
+        msg.attach_alternative(html_content, "text/html")
+        response = msg.send()
+        if response:
+            log.info (
+                'Email has been sent from "%s" to "%s" for content %s.',
+                from_email,
+                dest_emails,
+                content
+            )
+        )
+        return response
```

Comment:
I believe in this case, the response won't be boolean

Reply:
The return value will be the number of successfully delivered messages (which can be 0 or 1 since it can only send one message).